kubernetes/notebooks/lab-bench.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           mkdir /workspace/images
           cp -r /tmp/repo/demos/rag_agentic/src/utils.py /workspace/src/
           cp -r /tmp/repo/demos/rag_agentic/src/client_tools.py /workspace/src/
-          cp -r /tmp/repo/demos/rag_agentic/images/slack-message.png /workspace/images/slack-message.png
+          cp -r /tmp/repo/demos/rag_agentic/notebooks/images/slack-message.png /workspace/images/slack-message.png
```

Comment:
I don't think this line is necessary since everything in `notbooks/*` is pulled in in line 34.

Reply:
Correct. I will remove this line. Thanks.